cmd/prepare-storage.go

```diff
@@ -291,12 +291,12 @@ func waitForFormatErasure(firstDisk bool, endpoints Endpoints, poolCount, setCou
 	tries++ // tried already once
 
 	// Wait on each try for an update.
-	ticker := time.NewTicker(150 * time.Millisecond)
+	ticker := time.NewTicker(1 * time.Second)
 	defer ticker.Stop()
 
 	for {
 		select {
-		case <-ticker.C:
+		default:
```

Comment:
To avoid the initial 1 second wait before loading format.json from all drives, we can do something like this,
```
for { // for retries
   connectLoadInitFormats()
   select {
     case <-ticker.C:
     case <-globalOSSignalCh:
   }
}
```